blueprints/zesty-deploy-config/files/.circleci/config.yml

```diff
@@ -0,0 +1,123 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6.14-browsers
+    environment:
+      - TZ: "/usr/share/zoneinfo/America/Los_Angeles"
+    steps:
+        - checkout
+
+        # Install global dependencies
+        - run:
+            name: Install global dependencies
+            command: |
+              sudo npm install -g yarn@rc
+              <% if (bowerRequired) { %>sudo npm install -g bower<% } %>
```

Comment:
I am relieved that ember-cli makes conditionals in blueprints a straightforward thing.